setup.py

```
@@ -0,0 +1,23 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="skriptpy",
+    version="0.1.0",
+    author="ArchooD2",
+    author_email="archoo@example.com",
+    description="A bidirectional transpiler between Python and Skript",
+    long_description=open("README.md").read(),
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)